src/task7/task7

```diff
@@ -0,0 +1,34 @@
+function arrayToList(arr) {
+    let list = {};
+    for (let i = arr.length - 1; i >= 0; i--) {
+        list.value = arr[i];
+        list.rest = arr.length > 1 ? arrayToList(arr.slice(1)) : null;
+    }
+    return list;
+}
+
+function listToArray(list) {
+    let array = [];
+    for (; list; list = list.rest) {
```

Comment:
You are changing outer list value someone gave you.
This function should transform list into some array.
Not changing initial value.

Reply:
ok, got it